DirectEmployers/MyJobs
# [PD 2896] app access activities UI

## What's this PR do?
PR will hide and show things depending on the app access and activities for a current user

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull down the repository. If you have the ability to switch the activities for certain Users or yourself you can do that and switch back and forth to see certain buttons and such showing and hiding depending on the level of access that you have

## Any background context you want to provide?
